docs/admin-guide/run-plone.md

```diff
@@ -101,3 +101,12 @@ pip:
     ```
 
 For any of these commands, press {kbd}`ctrl-d` to stop the process.
+
+### Set a "fake" request
+
+To make sure that the request is fully set up for any code that uses `zope.globalrequest.getRequest`, you might need to use the following code.
+
+```python
+from zope.globalrequest import setRequest
+setRequest(app.REQUEST)
```

Comment:
app.REQUEST is not set up properly in the debug console:
```
>>> app.REQUEST
'<Special Object Used to Force Acquisition>'
```

So before you call setRequest, you have to also do:
```
from Testing.makerequest import makerequest
app = makerequest(app)
```